lib/participant.js

```diff
@@ -131,6 +172,16 @@ class Participant extends EventEmitter {
     ];
   }
 
+  /**
+   * @private
+   */
+  _getTrackPublicationEvents() {
+    // NOTE(mmalavalli): 'trackDisabled' and 'trackEnabled' will be
+    // re-emitted from the RemoteTrackPublication instead of RemoteTrack
+    // in twilio-video.js@2.0.0 onwards.
+    return [];
+  }
```

Comment:
I'm not sure how to understand this comment in this context. Can you clarify? Is it that LocalParticipant/RemoteParticipant will include these events when overriding `_getTrackPublicationEvents`?

Reply:
@markandrus Right now, Track's `enabled` and `disabled` are being re-emitted on the Participant as `trackEnabled` and `trackDisabled`. Ideally the Participant should re-emit `trackEnabled` and `trackDisabled` from the RemoteTrackPublication in 2.0.0+, since these events need to be fired even when the RemoteTrack is not subscribed to.